fly_qgc_mission error loading shared libraries: libjson11.so

```
/DroneCore/example/fly_qgc_mission/build$ ./fly_qgc_mission 
./fly_qgc_mission: error while loading shared libraries: libjson11.so: cannot open shared object file: No such file or directory
``

But building this example on macOS fails:
```
make
Scanning dependencies of target fly_qgc_mission
[ 50%] Building CXX object CMakeFiles/fly_qgc_mission.dir/fly_qgc_mission.cpp.o
[100%] Linking CXX executable fly_qgc_mission
Undefined symbols for architecture x86_64:
  "json11::Json::parse(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&, json11::JsonParse)", referenced from:
      dronecore::MissionImpl::import_qgroundcontrol_mission(std::__1::vector<std::__1::shared_ptr<dronecore::MissionItem>, std::__1::allocator<std::__1::shared_ptr<dronecore::MissionItem> > >&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in libdronecore_mission.a(mission_impl.cpp.o)
  "json11::Json::array_items() const", referenced from:
      dronecore::MissionImpl::import_mission_items(std::__1::vector<std::__1::shared_ptr<dronecore::MissionItem>, std::__1::allocator<std::__1::shared_ptr<dronecore::MissionItem> > >&, json11::Json const&) in libdronecore_mission.a(mission_impl.cpp.o)
  "json11::Json::number_value() const", referenced from:
      dronecore::MissionImpl::import_mission_items(std::__1::vector<std::__1::shared_ptr<dronecore::MissionItem>, std::__1::allocator<std::__1::shared_ptr<dronecore::MissionItem> > >&, json11::Json const&) in libdronecore_mission.a(mission_impl.cpp.o)
  "json11::Json::int_value() const", referenced from:
      dronecore::MissionImpl::import_mission_items(std::__1::vector<std::__1::shared_ptr<dronecore::MissionItem>, std::__1::allocator<std::__1::shared_ptr<dronecore::MissionItem> > >&, json11::Json const&) in libdronecore_mission.a(mission_impl.cpp.o)
  "json11::Json::operator[](std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) const", referenced from:
      dronecore::MissionImpl::import_mission_items(std::__1::vector<std::__1::shared_ptr<dronecore::MissionItem>, std::__1::allocator<std::__1::shared_ptr<dronecore::MissionItem> > >&, json11::Json const&) in libdronecore_mission.a(mission_impl.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [fly_qgc_mission] Error 1
make[1]: *** [CMakeFiles/fly_qgc_mission.dir/all] Error 2
make: *** [all] Error 2
```

--- a/example/fly_mission/CMakeLists.txt
+++ b/example/fly_mission/CMakeLists.txt
@@ -19,5 +19,4 @@ target_link_libraries(fly_mission
     dronecore_action
     dronecore_mission
     dronecore_telemetry
-    json11
 )

--- a/example/fly_qgc_mission/CMakeLists.txt
+++ b/example/fly_qgc_mission/CMakeLists.txt
@@ -18,6 +18,5 @@ target_link_libraries(fly_qgc_mission
     dronecore
     dronecore_action
     dronecore_mission
-    json11 # Needed by dronecore_mission for QGC mission plan import
     dronecore_telemetry
 )

--- a/example/fly_qgc_mission/CMakeLists.txt
+++ b/example/fly_qgc_mission/CMakeLists.txt
@@ -18,5 +18,6 @@ target_link_libraries(fly_qgc_mission
     dronecore
     dronecore_action
     dronecore_mission
+    json11 # Needed by dronecore_mission for QGC mission plan import
     dronecore_telemetry
 )

--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -3,7 +3,7 @@ include_directories(
     SYSTEM ${CMAKE_SOURCE_DIR}/third_party/mavlink/include
 )
 
-if(IOS OR ANDROID OR APPLE)
+if(IOS OR ANDROID)
     set(PLUGIN_LIBRARY_TYPE STATIC)
 else()
     set(PLUGIN_LIBRARY_TYPE SHARED)

